src/components/forms/InputField.js

```diff
@@ -322,6 +362,22 @@ class InputField extends PureComponent<Props> {
       return <TextareaWithAutosize {...props} />
     }
 
+    if (type === 'date') {
+      return <div className={`${dateInput}`} >
```

Comment:
This can be written simply as:
```suggestion
      return <div className={dateInput}>
```